finance-api/api-reference.md

```diff
@@ -0,0 +1,335 @@
+|INVOICE_PAYMENT |An invoice payment. It's only supported in Sweden. If an invoice is paid through a card payment, the payment type is `CARD_PAYMENT`. |Liquid, preliminary
+|INVOICE_PAYMENT_FEE |An invoice payment fee. It's only supported in Sweden. If an invoice is paid through a card payment, the payment fee type is `CARD_PAYMENT_FEE`. |Liquid, preliminary
+|PAYMENT |An alternative third-party payment method where Zettle handles the funds. For example, PayPal QR code and Klarna QR code. Contains a reference to the payment in the Purchase API. |Liquid, preliminary
+|PAYMENT_FEE |The fee for a third-party payment method. For example, PayPal QR code and Klarna QR code. Contains a reference to the payment fee in the Purchase API. |Liquid, preliminary
+|PAYOUT |A payout of the account balance from the merchant's liquid account to the merchant’s bank account. If the merchant is a PayPal user, the payout will be made to their PayPal Wallet. <br/>If the merchant's configuration has a minimum account balance, the payout is the liquid account balance minus the minimum account balance. For example, if the account balance is £147 and the minimum account balance is £47, the payout is £100. |Liquid
+|TELL_FRIEND (Deprecated) |Not applicable. |Not applicable 
+</details>
+
+
+### Responses
+<details open="true">
+<summary>Click to hide HTTP status codes.</summary>
+
+|Status code |Description
+|:---- |:----
+|200 OK |Returned when the operation is successful.  
+|400 Bad Request |Returned when a required parameter is missing or in a wrong format in the request.
+|401 Unauthorized |Returned when one of the following occurs: <br/><ul><li> The authentication information is missing in the request.</li><li>The authentication token has expired.</li><li>The authentication token is invalid.</li></ul>
+|403 Forbidden |Returned when the scope being used in the request is incorrect. 
+</details>
+
+<details open="true">
+<summary>Click to hide response attributes.</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |an array of objects|A list of transactions for the given account in the descending order that shows the most recent transaction first.
+|timestamp |string |Time when a transaction happens in the merchant's Zettle account. It's not the timestamp when a card transaction or a purchase happens. <br/>For example, transactions in the merchant's liquid account may happen hours after card transactions or purchases take place.
+|amount |integer |The amount of money of a transaction. If the transaction is a refund or a payment fee, the amount is negative.
```

Comment:
`If the transaction is a refund or a payment fee, the amount is negative`
There are more cases where transactions can have negative value, depending on the account type or transaction type. Suggestion: saying something like "Note that depending on the account type and transaction type, the amount can have a negative value".